avro-converter/src/test/java/io/confluent/connect/avro/AvroDataTest.java

```diff
@@ -687,6 +798,32 @@ public void testToConnectMapNonStringKeys() {
                  avroData.toConnectData(avroSchema, Arrays.asList(record)));
   }
 
+  /*
```

Comment:
It did, because it was just a copy of the previous test case :-)
I've implemented it now (and cleaned up some duplicate stuff) in AvroDataTest.